frame/contracts/src/exec.rs

```diff
@@ -35,8 +35,14 @@ use frame_support::{
 use frame_system::RawOrigin;
 use pallet_contracts_primitives::ExecReturnValue;
 use smallvec::{Array, SmallVec};
-use sp_core::ecdsa::Public as ECDSAPublic;
-use sp_io::{crypto::secp256k1_ecdsa_recover_compressed, hashing::blake2_256};
+use sp_core::{
+	ecdsa::Public as ECDSAPublic,
+	sr25519::{Public as SR25519Public, Signature as SR25519Signature},
+};
+use sp_io::{
+	crypto::{secp256k1_ecdsa_recover_compressed, sr25519_verify as crypto_sr25519_verify},
```

Comment:
I don't think we need the rename. Our function is scoped within the trait.

```suggestion
	crypto::{secp256k1_ecdsa_recover_compressed, sr25519_verify},
```